packages/next/src/bundles/webpack/packages/HotModuleReplacement.runtime.js

```diff
@@ -293,11 +293,10 @@ module.exports = function () {
 						return waitForBlockingPromises(function () {
 							if (applyOnUpdate) {
 								return internalApply(applyOnUpdate);
-							} else {
```

Comment:
Let's revert these stylistic refactors to keep the git-blame tidy

Reply:
Is a result from `ncc compiled`.